### ISSUE TITLE
Calling DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call DidYouMean.correct_error(error_name, spell_checker)' instead.

### ISSUE THREAD

**Issue by zchrb** [ruby/did_you_mean#170]
### Steps to reproduce
Running `rails -v` in terminal shows this warning:

```ruby
# Your reproduction script goes here
$ rails -v
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Rails 7.0.0
```
This is only occurring with `did_you_mean 1.6.1`.

### Expected behavior

No warning message. Should only show version number.

### Actual behavior

Version number shows with warning message.

### System configuration
**Rails version**: 7.0.0

**Ruby version**: 3.1.0

**did_you_mean version**: 1.6.1

**Comment by yahonda** [comment]
This deprecation warning should be removed once https://github.com/rails/thor/pull/761 has been merged to thor.

**Comment by yahonda** [comment]
thor v1.2.0 has been released which includes rails/thor#761 . I think this issue can be closed.
